Jenkinsfile

```diff
@@ -19,9 +20,8 @@ def deploy(environment) {
 pipeline {
   agent any
   environment {
-    DOCKER_IMAGE = "fixxx/looplijsten"
     APP = "looplijsten-api"
-    DOCKER_REGISTRY = "repo.secure.amsterdam.nl"
+    REPO_IMAGE = "${DOCKER_REGISTRY_NO_PROTOCOL}/fixxx/looplijsten"
```

Comment:
`REPO_IMAGE` is a slightly confusing name for me, suggesting it is an image of a repository or something. Should we perhaps name this `IMAGE_URL` or even `DOCKER_IMAGE_URL`?

Reply:
Changed!